src/App.js

```diff
@@ -1,9 +1,28 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from "react";
+import "./App.css";
+import Form from "./components/form";
+import TodoList from "./components/TodoList";
+
+import { createContext } from "react";
+export const ThemeContext = createContext("null");
 
 function App() {
+  const [inputText, setInputText] = useState("");
```

Comment:
Do you really need this state? you are only using it to set the value in the form to be `""`.  Why not init the form value to `""` ?